### PR TITLE
replacement "`" char

### PR DESCRIPTION
**Pull request by AliveMen**
replace   `   char to  "   for IE 11

### CODE REVIEW

File: VirtoCommerce.Platform.Web/Scripts/common/directives/chars-count.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
-﻿angular.module('platformWebApp')
+angular.module('platformWebApp')
     .directive('vaCharsCount', function () {
         return {
-            template: `<div class="chars-counter">
-                            <span>{{!!vaField ? vaField.length : 0}}</span> / <span>{{vaMaxlength}}</span> 
-                       </div>`,
+            template: "<div class=\"chars-counter\">"+
```
Comment (artem-dudarev) [inline VirtoCommerce.Platform.Web/Scripts/common/directives/chars-count.js]: Why not `'`?
Fix formatting.

Comment (AliveMen) [reply VirtoCommerce.Platform.Web/Scripts/common/directives/chars-count.js]: done

File: VirtoCommerce.Platform.Web/Scripts/common/directives/chars-count.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
-﻿angular.module('platformWebApp')
+angular.module('platformWebApp')
     .directive('vaCharsCount', function () {
         return {
-            template: `<div class="chars-counter">
-                            <span>{{!!vaField ? vaField.length : 0}}</span> / <span>{{vaMaxlength}}</span> 
-                       </div>`,
+            template: '<div class="chars-counter">' +
```
Comment (artem-dudarev) [inline VirtoCommerce.Platform.Web/Scripts/common/directives/chars-count.js]:
```javascript
template: '<div class="chars-counter"> \
              <span>{{!!vaField ? vaField.length : 0}}</span> / <span>{{vaMaxlength}}</span> \
          </div>',
```

Comment (AliveMen) [reply VirtoCommerce.Platform.Web/Scripts/common/directives/chars-count.js]: done

**Pull request merged**